pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/executor/LeafSchedulerService.java

```diff
@@ -31,17 +34,20 @@
 import org.slf4j.LoggerFactory;
 
 
-public class LeafSchedulerService {
+public class LeafSchedulerService implements SchedulerService {
```

Comment:
where is the SchedulerService for opchain? I only see leafscheduler

Reply:
this is the scheduler for OpChain. both leaf and intermediate are modeled as opchain going forward there will not be a separate model

Reply:
then lets rename it to OpChainSchedulerService